UEFITool/uefitool.pro

```diff
@@ -8,6 +8,8 @@ DEFINES += "U_ENABLE_NVRAM_PARSING_SUPPORT"
 DEFINES += "U_ENABLE_ME_PARSING_SUPPORT"
 DEFINES += "U_ENABLE_GUID_DATABASE_SUPPORT"
 
+greaterThan(QT_MAJOR_VERSION, 5): DEFINES += "QT_VERSION_MAJOR=6"
```

Comment:
Are you sure there is no builtin macro to check against Qt version?

Reply:
True.
This isn't needed.
Pushed commit